react/features/toolbox/components/VideoMuteButton.js

```diff
@@ -160,8 +160,7 @@ class VideoMuteButton extends AbstractVideoMuteButton<Props, *> {
     _setVideoMuted(videoMuted: boolean) {
         sendAnalytics(createToolbarEvent(VIDEO_MUTE, { enable: videoMuted }));
         if (this.props._audioOnly) {
-            this.props.dispatch(
-                setAudioOnly(false, /* ensureTrack */ true));
```

Comment:
`setVideoMuted` action is dispatched right after which creates the track (if the track doesn't exist) when camera is unmuted using the mute button. Earlier it was a no-op since  setAudioOnly action created the track and unmuted the video - https://github.com/jitsi/jitsi-meet/blob/master/react/features/toolbox/components/VideoMuteButton.js#L168